packages/serverless/src/backend_connector.js

```diff
@@ -308,6 +308,9 @@ function send(resourcePath, payload, finalLambdaRequest, callback) {
       if (finalLambdaRequest) {
         req.removeAllListeners();
         req.on('error', () => {});
+        // At this point we already received a response from the server, but since we removed all listeners we have to
+        // manually clean up the request.
+        req.destroy();
```

Comment:
Changes in the backend connector necessitate a unit test. Could you please add a test for the bug and its corresponding solution?

Reply:
I had a look and tried different options, but then realized that the problem can't be reproduced with the test setup here. Since no real requests are send, but a sinon stub is used, we don't see any unclosed / leaking sockets.

Now knowing that the req must be destroyed I can check hat req.destroy actually gets called

Reply:
Thank you for adding the case. We'll consider adding the integration test later.